src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setColor(color);

```

Comment:
you should use a constructor of the parent to initialize this field, the same for other figures